lib/services/messenger/translators/translator-scaffold.ts

```diff
@@ -74,10 +74,10 @@ export abstract class TranslatorScaffold implements Translator {
 				const charRegex = new RegExp(`^${charCapture}`, 'im');
 				return TranslatorScaffold.metadataByRegex(message, charRegex);
 			case MetadataEncoding.HiddenHTML:
-				const hiddenHTMLRegex = new RegExp(`^<a href="${baseUrl}${querystring}"[^>]*></a>`, 'im');
+				const hiddenHTMLRegex = new RegExp(`<a href="${baseUrl}${querystring}"[^>]*></a>$`, 'i');
```

Comment:
Unit tests to make sure we never have regressions or security issues in the future :)

Reply:
Done